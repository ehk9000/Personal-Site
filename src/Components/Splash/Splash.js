import React from 'react';


const Splash = () => {
  <main>
    <img src="" alt=""/>

  </main>
}

export default Splash;