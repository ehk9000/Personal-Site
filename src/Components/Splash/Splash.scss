main {
  display: flex;
  flex-flow: column;
  align-items: center;
  height: 80vh;
  width: auto;

  img {
    position: relative;
    z-index: -1;
    height: 60vh;
    width: auto;
  }

  h3 {
    margin: 8px auto;
  }
}

.bio {
  display: flex;
  flex-flow: column;
  text-align: center;
  font-size: 18px;
  color: #565264;
  padding: 25px;
  width: auto

  p {
    margin-top: 10px;
  }
}
