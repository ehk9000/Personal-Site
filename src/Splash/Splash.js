import React from 'react';

const Splash = () => {
  <main>
    
  </main>
}

export default Splash;