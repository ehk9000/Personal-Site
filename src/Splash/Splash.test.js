import React from 'react';
import { shallow } from 'enzyme';
import  Splash  from './Splash';

describe(('Splash'), () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Splash />)
  });

  it('Should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});