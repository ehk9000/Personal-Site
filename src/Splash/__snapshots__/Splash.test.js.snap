// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash Should match snapshot 1`] = `ShallowWrapper {}`;
